packages/web/components/Technology/Details/ReadinessLevel/index.js

```diff
@@ -10,9 +11,8 @@ const ReadinessLevel = () => {
 	const [imageHeight, setImageHeight] = useState(0);
 	const [levelHeight, setLevelHeight] = useState(0);
 
-	const {
-		technology: { taxonomies: { readinessLevel: { slug = '' } = {} } = {} } = {},
-	} = useTechnology();
+	const { technology: { terms = [] } = {} } = useTechnology();
+	const { slug } = normalizeTrl(terms);
```

Comment:
Acho que assim fica mais lindo :sweat_smile: 
```suggestion
	const { technology } = useTechnology();
	const { slug } = normalizeTrl(technology.terms);
```

Reply:
O default assign era pra garantir que se algo vier `undefined` não vá quebrar as interações. O que acha?

Reply:
Acho que então seria bacana um optional chaining e lá na função de normalize ter a checassem se existe algo ou não.
```suggestion
	const { technology } = useTechnology();
	const { slug } = normalizeTrl(technology?.terms);
```